cpp/code/CMakeLists.txt

```diff
@@ -13,12 +13,21 @@ if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
     set(CMAKE_CXX_CLANG_TIDY "clang-tidy")
 endif()
 
+# Add protobuf
+find_package(Protobuf REQUIRED)
+find_package(gRPC CONFIG REQUIRED)
+find_package(Threads)
+
+set(PROTO_FILES
+    protos/helloworld.proto
+)
+
 # Create test targets
 enable_testing()
 
 include(GoogleTest)
 
-function(RECIPE TARGET)
+function(RECIPE TARGET COMPILE_PROTOS)
```

Comment:
This is probably good enough for now and maybe we can add keyword arguments eventually.

Though, would it work to just drop 
```
target_link_libraries(flight ...)
...
protobuf_generate(TARGET flight LANGUAGE ...)
...
```
below without adding a parameter?

Reply:
Ah I can move it after 👍